actioncable/lib/action_cable/engine.rb

```diff
@@ -24,11 +24,11 @@ class Railtie < Rails::Engine # :nodoc:
       options = app.config.action_cable
       options.allowed_request_origins ||= "http://localhost:3000" if ::Rails.env.development?
 
-      app.paths.add "config/redis/cable", with: "config/redis/cable.yml"
+      app.paths.add "config/cable", with: "config/cable.yml"
 
       ActiveSupport.on_load(:action_cable) do
-        if (redis_cable_path = Pathname.new(app.config.paths["config/redis/cable"].first)).exist?
-          self.redis = Rails.application.config_for(redis_cable_path).with_indifferent_access
+        if (config_path = Pathname.new(app.config.paths["config/cable"].first)).exist?
```

Comment:
To this work the file need to be registered. https://github.com/rails/rails/blob/48f5cecdb03f697a710169ccef1885fcaca70788/actioncable/lib/action_cable/engine.rb#L27